app/org/sagebionetworks/bridge/dao/AccountDao.java

```diff
@@ -122,4 +122,12 @@
      *      about the request and the total number of records.
      */
     public PagedResourceList<AccountSummary> getPagedAccountSummaries(Study study, int offsetBy, int pageSize, String emailFilter);
+    
+    /**
+     * For MailChimp, and other external systems, we need a way to get a healthCode for a given email.
+     * @param study
+     * @param email
+     * @return
+     */
```

Comment:
Minor: We don't need the @param and @return in the comments if we aren't actually going to fill them in.